incubator/node/README.md

```diff
@@ -0,0 +1,87 @@
+# node
+
+[Node](https://www.nodejs.org) Event-driven I/O server-side JavaScript environment based on V8
+
+## TL;DR;
+
+```console
+$ helm install .
+```
+
+## Introduction
+
+This chart bootstraps a [Node](https://github.com/bitnami/bitnami-docker-node) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+It clones and deploy a nodejs application  from a git repository and  defined revision.
```

Comment:
It clones and deploy**s** a Node.js application from a git repository.

(I think we can omit the part about the revision)